test/dotnet-new3.UnitTests/CommonTemplatesTests.cs

```diff
@@ -144,35 +125,29 @@ public void AllCommonProjectsCreateRestoreAndBuild(string expectedTemplateName,
         [InlineData("Console App", "console", "C#")]
         [InlineData("Console App", "console", "F#")]
         [InlineData("Console App", "console", "VB")]
-        [InlineData("Console App", "console", "C#", "net6.0")]
-        [InlineData("Console App", "console", "F#", "net6.0")]
-        [InlineData("Console App", "console", "VB", "net6.0")]
-        [InlineData("Console App", "console", "C#", "net5.0")]
-        [InlineData("Console App", "console", "F#", "net5.0")]
-        [InlineData("Console App", "console", "VB", "net5.0")]
-        [InlineData("Console App", "console", "C#", "netcoreapp3.1")]
-        [InlineData("Console App", "console", "F#", "netcoreapp3.1")]
-        [InlineData("Console App", "console", "VB", "netcoreapp3.1")]
-        [InlineData("Console App", "console", "C#", "netcoreapp2.1")]
-        [InlineData("Console App", "console", "F#", "netcoreapp2.1")]
-        [InlineData("Console App", "console", "VB", "netcoreapp2.1")]
+        [InlineData("Console App", "console", "C#", "net7.0")]
+        [InlineData("Console App", "console", "F#", "net7.0")]
+        [InlineData("Console App", "console", "VB", "net7.0")]
+        [InlineData("Console Application", "console", "C#", "net5.0")]
+        [InlineData("Console Application", "console", "F#", "net5.0")]
+        [InlineData("Console Application", "console", "VB", "net5.0")]
+        [InlineData("Console Application", "console", "C#", "netcoreapp3.1")]
+        [InlineData("Console Application", "console", "F#", "netcoreapp3.1")]
+        [InlineData("Console Application", "console", "VB", "netcoreapp3.1")]
```

Comment:
Why did we remove .net 6 tests?

Reply:
Because I believe net5.0 and net7.0 are big enough sample, 3.1 is staying because its netcoreapp sample... Notice that we need to reduce number of integrations tests, ideally we move 99% of this tests to installer, where they matter...